tests/src/python/test_qgsserver_wms_getmap.py

```diff
@@ -1806,6 +1806,25 @@ def test_wms_getmap_tile_buffer(self):
         r, h = self._result(self._execute_request(qs))
         self._img_diff_error(r, h, "WMS_GetMap_Tiled_Labels_True")
 
+        # Check with rotated labels and tiled=true
+        qs = "?" + "&".join(["%s=%s" % i for i in list({
+            "MAP": urllib.parse.quote(
+                os.path.join(self.testdata_path, 'wms_tile_buffer.qgs')),
+            "SERVICE": "WMS",
+            "VERSION": "1.3.0",
+            "REQUEST": "GetMap",
+            "BBOX": "317654,6163276,327603,6173244",
+            "CRS": "EPSG:3857",
+            "WIDTH": "512",
+            "HEIGHT": "512",
+            "LAYERS": "wms_tile_buffer_labels_rotated",
+            "FORMAT": "image/png",
+            "TILED": "true"
+        }.items())])
+
+        r, h = self._result(self._execute_request(qs))
+        self._img_diff_error(r, h, "WMS_GetMap_Tiled_Rotated_Labels_True")
```

Comment:
The text in the reference image is quite narrow, so the test will be rather flaky on different platforms. Can you use the reference test font (QGIS Vera sans) in bold and bump the font size here?